docker/fixtures/index.ts

```diff
@@ -31,90 +31,106 @@ export const cosmosdbClient = new CosmosClient({
+    isVisible: true,
+    maxAllowedPaymentAmount: 100000,
+    organizationFiscalCode: "01234567890",
+    organizationName: "Organization name",
+    requireSecureChannels: false,
+    serviceId: config.REQ_SERVICE_ID,
+    serviceName: "MyServiceName"
+  }),
+  E.getOrElse(() => {
+    throw new Error("Cannot decode service payload.");
+  })
+)
 
-const aNewService = NewService.decode({
-  ...aService,
-  kind: "INewService"
-}).getOrElseL(() => {
-  throw new Error("Cannot decode new service.");
-});
+const aNewService = pipe(
+  NewService.decode({
+    ...aService,
+    kind: "INewService"
+  }),
+  E.getOrElse(() => {
+    throw new Error("Cannot decode new service.");
+  }))
 
-const aProfile: Profile = Profile.decode({
-  acceptedTosVersion: 1,
-  email: "email@example.com",
-  fiscalCode: "AAAAAA00A00A000A",
-  isEmailEnabled: true,
-  isEmailValidated: true,
-  isInboxEnabled: true,
-  isWebhookEnabled: true
-}).getOrElseL(() => {
-  throw new Error("Cannot decode profile payload.");
-});
+const aProfile: Profile = pipe(
+  Profile.decode({
+    acceptedTosVersion: 1,
+    email: "email@example.com",
+    fiscalCode: "AAAAAA00A00A000A",
+    isEmailEnabled: true,
+    isEmailValidated: true,
+    isInboxEnabled: true,
+    isWebhookEnabled: true
+  }),
+  E.getOrElse(() => {
+    throw new Error("Cannot decode profile payload.");
+  }))
 
-const aNewProfile = NewProfile.decode({
-  ...aProfile,
-  kind: "INewProfile"
-}).getOrElseL(() => {
-  throw new Error("Cannot decode new profile.");
-});
+const aNewProfile = pipe(
+  NewProfile.decode({
+    ...aProfile,
+    kind: "INewProfile"
+  }), E.getOrElse(() => {
+    throw new Error("Cannot decode new profile.");
+  })
+)
 
-createDatabase(config.COSMOSDB_NAME)
-  .chain(db =>
-    sequenceT(taskEitherSeq)(
-      createCollection(db, "message-status", "messageId"),
-      createCollection(db, "messages", "fiscalCode"),
-      createCollection(db, "notification-status", "notificationId"),
-      createCollection(db, "notifications", "messageId"),
-      createCollection(db, "profiles", "fiscalCode"),
-      createCollection(db, "services", "serviceId")
-    ).map(_ => db)
-  )
-  .chain(db =>
-    sequenceT(taskEitherSeq)(
-      new ServiceModel(db.container(SERVICE_COLLECTION_NAME)).create(
-        aNewService
+pipe(
+  createDatabase(config.COSMOSDB_NAME),
+  TE.chain(db =>
+    pipe(
+      sequenceT(TE.ApplySeq)(
+        createCollection(db, "message-status", "messageId"),
+        createCollection(db, "messages", "fiscalCode"),
+        createCollection(db, "notification-status", "notificationId"),
+        createCollection(db, "notifications", "messageId"),
+        createCollection(db, "profiles", "fiscalCode"),
+        createCollection(db, "services", "serviceId")
       ),
-      new ProfileModel(db.container(PROFILE_COLLECTION_NAME)).create(
-        aNewProfile
-      )
-    ).mapLeft(_ => new Error(`CosmosError: ${_.kind}`))
+      TE.map(_ => db)
+    )
+  ),
```

Comment:
TE.chainFirst(db =>
      sequenceT(TE.ApplySeq)(
        createCollection(db, "message-status", "messageId"),
        createCollection(db, "messages", "fiscalCode"),
        createCollection(db, "notification-status", "notificationId"),
        createCollection(db, "notifications", "messageId"),
        createCollection(db, "profiles", "fiscalCode"),
        createCollection(db, "services", "serviceId")
      )
  )

you could replace it with a TE.chainFirst